ps-modules/MacUtil/MacUtil.psm1

```diff
@@ -0,0 +1,202 @@
+        }
+        elseif (-not $item.PSIsContainer) {
+            Write-Host "> Processing: $filename - Running lipo"
+            Invoke-Expression -Command "lipo -create -output `"$destPath`" `"$srcPathArm64`" `"$srcPathX64`""
+        }
+    }
+}
+
+function Update-LibraryPath {
+    param (
+        [string]$libPath
+    )
+
+    Write-Host ">> Update-LibraryPath..."
+
+    if (-not (Test-Path $libPath -PathType Leaf)) {
+        Write-Host ">> File '$libPath' does not exist."
+        return
+    }
+
+    Write-Host ">> Updating library path for: $libPath..."
+    #$otoolCmd = "otool -L $libPath"
+    #$otoolOutput = Invoke-Expression -Command $otoolCmd
+    $lines = $otoolOutput.Split([Environment]::NewLine)
+    foreach ($line in $lines) {
+        if (-not $line.StartsWith("	")) {
+            continue
+        }
+
+        $path = $line.Substring(1, $line.IndexOf('(') - 1).Trim()
+        $dirname = [System.IO.Path]::GetDirectoryName($path)
+
+        if ($dirname -match "/usr/lib" -or $dirname -match "/System") {
+            continue
+        }
+
+        $oldPathLoc = $path.IndexOf($dirname)
+
+        if ($oldPathLoc -eq -1) {
+            continue
+        }
+
+        $newPath = $path.Replace($dirname, "@rpath")
+        $dylibName = [System.IO.Path]::GetFileName($libPath)
+
+        Write-Host ($path + "-" + $dylibName)
+
+        if ([System.IO.Path]::GetFileName($path) -eq $dylibName) {
+            Write-Host ">>> Changing ID"
+            $changeCmd = "install_name_tool -id $newPath $libPath"
+        }
+        else {
+            Write-Host ">>> Changing path"
+            $changeCmd = "install_name_tool -change $path $newPath $libPath"
+        }
+
+        Invoke-Expression -Command $changeCmd
+    }
+
+    #Write-Host ">> Final result"
+    #Invoke-Expression -Command $otoolCmd
+}
+
+function ConvertTo-RelativeInstallPaths {
+    param (
+        [string]$directory,
+        [string]$extension
+    )
+
+    $libFiles = Get-ChildItem -Path $directory -Filter "*.$extension"
+    foreach ($libFile in $libFiles) {
+        $filePath = $libFile.FullName
+        $fileName = $libFile.Name
+        Write-Host ">> Processing: $fileName"
+        Set-ItemProperty -Path $filePath -Name IsReadOnly -Value $false
+        Update-LibraryPath -libPath $filePath
+    }
+}
+
+Function Remove-DylibSymlinks {
+    param (
+        [Parameter(Mandatory=$true)][string]$BuildArtifactsPath
+    )
+
+    Push-Location "$BuildArtifactsPath/lib"
+
+    # Enumerate files
+    $main_files = @()
+    $files_with_versions = @()
+    $dylib_files = Get-ChildItem -Path . -Filter "*.dylib"
+    foreach ($dylib_file in $dylib_files) {
+        # Check if the file name contains more than one dot
+        if (-not ($dylib_file.Name -match '\..*\..*')) {
+            $main_files += $dylib_file.Name
+        }
+        else {
+            $files_with_versions += $dylib_file.Name
+        }
+    }
+
+    Write-Host ""
```

Comment:
If you want an extra new line before the next Write-Host, couldn't you just prepend the next Write-Host with a `\n`

Reply:
I could, but it doesn't look as pretty having two lines of "===", with one of them off by 2 😄.  I'll just refactor out the print-banner thing you mentioned earlier and be done with it.

Reply:
Fixed: 6e305d93edc6a8841d455d02013b239ed45d96c0